src/app/clusters/diagnostic-logs-server/diagnostic-logs-server.h

```diff
@@ -60,6 +63,12 @@ class DiagnosticLogsServer
 
 private:
     static DiagnosticLogsServer sInstance;
+
+    void InvokeCommand(HandlerContext & ctx) override;
+
+#ifdef MATTER_DM_DIAGNOSTIC_LOGS_CLUSTER_CLIENT_ENDPOINT_COUNT
```

Comment:
This is always defined now.

Reply:
It should be DIAGNOSTIC_LOGS_ENABLE_RETRIEVE_LOGS_REQUEST_CMD or none since this command is not optional, but I observed flash increase, so I tested MATTER_DM_DIAGNOSTIC_LOGS_CLUSTER_CLIENT_ENDPOINT_COUNT